notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>consider linking to <code>model</code> https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.models</p><p> </p><p>Also, the second sentence "You select ...." sounds like the user must select one when a suitable default has already been chose for the tutorial. Consider a different wording, perhaps: </p><p> </p><p> </p><p>This tutorial uses the PyTorch v1.11 container, but for your own use case, you can choose from the list of prebuilt PyTorch containers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>